src/commands/build-util.ts

```diff
@@ -49,9 +49,10 @@ export async function buildFile(
         const destBuildFile = output[destExt];
         const result = await step.transform({
           contents: destBuildFile,
+          extension: destExt,
           filePath: rootFileName + destExt,
           isDev,
-          log: (msg, data) => {
+          log: (msg, data = {}) => {
```

Comment:
I wanted to safeguard this a bit, as a plugin may call `log()` without `data` defined

Reply:
ah nice!